webapp-django/crashstats/crashstats/models.py

```diff
@@ -161,7 +161,96 @@ def inner(*args, **kwargs):
     return inner
 
 
+def memoize(function):
+    """Decorator for model methods to cache in memory or the filesystem
+    using CACHE_MIDDLEWARE and/or CACHE_MIDDLEWARE_FILES Django config"""
+
+    @functools.wraps(function)
+    def memoizer(*args, **kwargs):
+
+        def get_cached_result(key):
```

Comment:
Yes! That was an error in refactoring on my part, rather than intentional. Thanks for catching it.